tests/ui/UnreadIndicatorsTest.tsx

```diff
@@ -108,7 +119,7 @@ beforeAll(() => {
     // fetch() never gets called so it does not need mocking) or we might have fetch throw an error to test error handling
     // behavior. But here we just want to treat all API requests as a generic "success" and in the cases where we need to
     // simulate data arriving we will just set it into Onyx directly with Onyx.merge() or Onyx.set() etc.
-    global.fetch = TestHelper.getGlobalFetchMock();
+    global.fetch = TestHelper.getGlobalFetchMock() as typeof global.fetch;
```

Comment:
Maybe we should wait till `TestHelper` is migrated to TS, I mean use ts-expect-error instead of assertion. Will we need the assertion if `TestHelper` is migrated?